pennylane/qnn/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the :class:`~.KerasLayer` class for integrating QNodes with the Keras
+layer API."""
```

Comment:
Perhaps we should make this more general, this can probably be moved to `keras.py`.
```suggestion
"""This module contains classes and functions for constructing quantum neural networks from QNodes."""
```